sass/components/_main-menu.scss

```diff
@@ -92,11 +54,24 @@
   color: $color-primary;
 }
 
+.card-buttons {
+  position: absolute;
+  opacity: 0;
+  width: 100%;
+  bottom: 3rem;
+  color: $color-primary;
+  font-family: Montserrat;
```

Comment:
Consider to always provide fallback font family entries for cases when a file download would fail or would take longer.  
It's a good idea to provide a web safe fallback font.